guide/C/ch_reports.xml

```diff
@@ -613,4 +613,183 @@ an empty report will be generated.</para>
 </sect3>
 </sect2>
 </sect1>
+
+<sect1 id="rpt_transaction">
+  <title>Transaction Report</title>
+  <sect2 id="rpt_TR_new">
+    <title>New in Gnucash 3.0</title>
+    <para>The Transaction Report has undergone several major upgrades, bug fixes, and new features. It aims to be the most versatile report to retrieve useful information from the financial records.</para>
+    <itemizedlist>
+      <listitem>
+        <para><emphasis>New!</emphasis> Debit and Credit subtotals are now displayed in the appropriate columns. Within a grouping section, if debit totals > credit totals, the group subtotal will be displayed in the debit column.</para>
+      </listitem>
+      <listitem>
+        <para><emphasis>New!</emphasis> Multiple data columns enable display of foreign currency and original currency in separate columns.</para>
+      </listitem>
+      <listitem>
+        <para><emphasis>New!</emphasis> Additional sortkeys e.g. sort by reconcile status (unreconciled -> cleared -> reconciled), sort by weekday, allow additional views into the financial records.</para>
+      </listitem>
+      <listitem>
+        <para><emphasis>New!</emphasis> Full text filter enabled for description, notes and memo, and for account full names. This allows easier selection of source account names and tailored reporting. Filters can optionally use full POSIX regular expressions.</para>
+      </listitem>
+      <listitem>
+        <para><emphasis>New!</emphasis> Indenting of columns will indent the report columns for easier understanding of grouping and sorting. (This is now enabled by default)</para>
+      </listitem>
+      <listitem>
+        <para><emphasis>New!</emphasis> Subtotal table is useful to understand the relationship in the data e.g. account values in time series.</para>
+      </listitem>
+      <listitem>
+        <para><emphasis>New!</emphasis> Filter by Closing Transactions will disallow closing transactions (i.e. income &amp; expense accounts to equity) which should not routinely be included in transaction reports. (This is now enabled by default.)</para>
+      </listitem>
+      <listitem>
+        <para><emphasis>New!</emphasis> Options Summary will print a list of report options that were changed by the user. This is mainly useful to troubleshoot reports.</para>
+      </listitem>
+      <listitem>
+        <para>Bugs fixed - numerous bugs identified were fixed; e.g. sign display was fixed, correct grouping of periodic dates.</para>
+      </listitem>
+    </itemizedlist>
```

Comment:
While we do need  a "What's new" document, it's better to make it separate--at least its own chapter if not a page on the website--rather than including it into the body of the Guide where it will get stale in a year or two.

Reply:
Ok let me know where to move this part to?

Reply:
I think we need to discuss that on the dev list. My inclination is to put together a page for the website.